cns/NetworkContainerContract.go

```diff
@@ -481,6 +481,7 @@ type PodIpInfo struct {
 	SkipDefaultRoutes bool
 	// Routes to configure on interface
 	Routes []Route
+	PnPID  string
```

Comment:
i thought we decided to remove this change and keep as separate PR for cns change

Reply:
need this for CNI; since CNS PR has not merged yet, let's use this PR to add

Reply:
removed